### DISCUSSION TITLE
Within the Initial View within Document Properties, the default is set to 'Show File Name' - how do I change this

### DISCUSSION THREAD

**Discussion by gchickering21** [py-pdf/pypdf#1932] [Q&A]
Within the Initial View withinDocument Properties, the default is set to 'Show File Name', does this package provide a way to change it so it defaults to 'Show Document Title'. The documentation says this package has a way to change viewer settings but I am not seeing any way of doing this

**Comment by pubpub-zz** [comment]
to access the root object of a Writer Object, use `._root_object`
as the default state of this property is false, the easiest would be:
```
del writer._root_object["/ViewerPreferences"]["/DisplayDocTitle"]
```
